build.gradle

```diff
@@ -232,6 +239,12 @@ configurations {
 }
 
 dependencies {
+    implementation('net.minidev:json-smart:2.4.9') {
```

Comment:
Could you explain what these new dependencies are?

Reply:
This is used as transitive dependencies in one of the package. Overriding the newer version for the compatibles.